grammar.js

```diff
@@ -475,7 +520,6 @@ module.exports = grammar({
       $.tuple_literal,
       $.step_expr_or_stutter,
       $.step_expr_no_stutter,
-      $.fairness,
```

Comment:
Why is this removed?

Reply:
Fixed.
This accidental change was causing latest corpus parsing failures